stdlib/3/os/__init__.pyi

```diff
@@ -182,8 +182,24 @@ class statvfs_result:  # Unix only
     f_namemax = 0
 
 # ----- os function stubs -----
-def fsencode(filename: str) -> bytes: ...
-def fsdecode(filename: bytes) -> str: ...
+if sys.version_info >= (3, 6):
+    def fsencode(filename: Union[str, PathLike]) -> bytes: ...
```

Comment:
It actually also accepts bytes, and this is documented at https://docs.python.org/3/library/os.html#os.fsencode.